### PR TITLE
Allow work updating through API

### PR DESCRIPTION
**Pull request by kdid**
Adds `updateWork` mutation

### CODE REVIEW

File: assets/js/gql/FileSetFields.frag.gql [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  id
+  accessionNumber
+  role
+  rank
+  position
+  metadata {
+    description
+    digests
+  }
+}
```
Comment (adamjarling) [inline assets/js/gql/FileSetFields.frag.gql]: Are we going to keep using these `gql` files?  The front end (for now) is not using them...just wondering if housing in the `/assets/` folder is best place?

Comment (kdid) [reply assets/js/gql/FileSetFields.frag.gql]: We can move them. I'll open an issue to move them out of the front end.

Comment (mbklein) [reply assets/js/gql/FileSetFields.frag.gql]: Maybe to `/test/gql/` or `/test/fixtures/gql/`?

Comment (kdid) [reply assets/js/gql/FileSetFields.frag.gql]: @mbklein - can you add your thoughts to the issue?
https://github.com/nulib/repodev_planning_and_docs/issues/435

**Pull request merged**